dbconn/dbconn.go

```diff
@@ -257,6 +258,19 @@ func (dbconn *DBConn) MustExec(query string, whichConn ...int) {
 	gplog.FatalOnError(err)
 }
 
+func (dbconn *DBConn) ExecContext(queryContext context.Context, query string, whichConn ...int) (sql.Result, error) {
```

Comment:
`ExecQueryWithContext` might be a more appropriate function name here.

Reply:
I'd rather not differ from the actual Golang database/sql ExecContext function name.  This dbconn wrapper seems to be intended for doing a bit of extra GPDB-related stuff before running the actual function.  I wouldn't want to remember two function names.  If so, we should rename a lot of the functions in this file to be GPDB-specific instead (e.g. Exec -> GPDBExecQuery, Select -> GPDBSelectQuery, etc) to avoid possible confusion.